public/javascripts/task.js

```diff
@@ -100,25 +87,32 @@ const postForm = async (url, formData, httpMethod) => {
 };
 
 const createDelButton = (id) => {
-  const delButton = document.createElement("button");
-  delButton.classList.add("task-delete-btn");
+  const delButton = document.createElement("i");
+  delButton.classList.add("fas", "fa-trash-alt", "task-delete-btn");
   delButton.dataset.id = id;
-  delButton.innerHTML = "X";
-  return delButton;
+  const td = document.createElement("td");
+  td.append(delButton)
+  return td;
 };
 
-const createListGroup = (name, id) => {
-  const listGroup = document.createElement("div");
-  listGroup.classList.add("list-group");
-
-  const taskItem = document.createElement("li");
-  taskItem.innerHTML = name;
-  taskItem.id = `task-${id}`
-
-  listGroup.appendChild(taskItem);
-  taskItem.insertAdjacentElement("beforeend", createDelButton(id));
-
-  return listGroup;
+const fillTableCell = (data) => {
+  const td = document.createElement("td");
+  const p = document.createElement("p");
+  p.innerText = data;
+  td.append(p);
+  return td
+}
+
+const createTableRow = (task) => {

```

Comment:
Nice work moving the logic for table cell and delete button into their own methods!